### ISSUE TITLE
./kevm debug errors, saying "Multiple tool activations"

### ISSUE THREAD

**Issue by pirapira** [kframework/evm-semantics#160]
When I follow the readme, I see an error message from the `./kevm debug` invocation
```
$ ./kevm debug tests/ethereum-tests/VMTests/vmArithmeticTest/add0.json
== debugging: tests/ethereum-tests/VMTests/vmArithmeticTest/add0.json
[Error] Critical: Multiple tool activations found: --interpret and --debugger
```

**Comment by ehildenb** [comment]
This is fixed in https://github.com/kframework/evm-semantics/pull/153

**Comment by ehildenb** [comment]
Fixed in commit 04e332e16
